plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/AbstractTestIcebergConnectorTest.java

```diff
@@ -203,6 +209,46 @@ public void testShowCreateTable()
                         ")");
     }
 
+    @Override
+    protected void checkInformationSchemaTablesForPointedQueryForMaterializedView(String schemaName, String viewName)
+    {
+        // TODO The query should not fail, obviously. It should return the viewName, as information_schema.tables returns it when invoked without filters
+        assertThatThrownBy(() -> super.checkInformationSchemaTablesForPointedQueryForMaterializedView(schemaName, viewName))
+                .hasMessageContaining("Not an Iceberg table");
+    }
+
+    @Override
+    protected void checkShowColumnsForMaterializedView(String viewName)
+    {
+        // TODO The query should not fail, obviously. It should return all the columns in the view
+        assertThatThrownBy(() -> super.checkShowColumnsForMaterializedView(viewName))
+                .hasMessageContaining("Not an Iceberg table");
+    }
+
+    @Override
+    protected void checkInformationSchemaColumnsForMaterializedView(String schemaName, String viewName)
+    {
+        // TODO The query should not fail, obviously. It should return columns for all tables, views, and materialized views
+        assertThatThrownBy(() -> super.checkInformationSchemaColumnsForMaterializedView(schemaName, viewName))
+                .hasMessageFindingMatch("(?s)Expecting.*to contain:.*, nationkey\\).*, name\\).*, regionkey\\).*, comment\\)");
+    }
+
+    @Override
+    protected void checkInformationSchemaColumnsForPointedQueryForMaterializedView(String schemaName, String viewName)
+    {
+        // TODO The query should not fail, obviously. It should return columns for the materialized view
+        assertThatThrownBy(() -> super.checkInformationSchemaColumnsForPointedQueryForMaterializedView(schemaName, viewName))
+                .hasMessageContaining("Not an Iceberg table");
+    }
+
+    @Override
+    protected void checkInformationSchemaViewsForMaterializedView(String schemaName, String viewName)
+    {
+        // TODO should return materialized view, as it's also a view
```

Comment:
`should return materialised view, as it's also a view`
I don't think we have established what is the right behaviour here
We might want to list MVs in another metadata table instead of listing in information_schema.views

Reply:
will add a comment

Reply:
```
// TODO should probably return materialized view, as it's also a view -- to be double checked